content/en/security_platform/cloud_siem/log_detection_rules.md

```diff
@@ -126,6 +134,37 @@ Anomaly detection inspects how the `group by` attribute has behaved in the past.
 
 **Note**: The query applies to all Datadog events and ingested logs that do not require indexing.
 
+{{% /tab %}}
+
+{{% tab "Impossible Travel" %}}
+
+### Search query
+
+Construct a search query using the same logic as a [log explorer search][1]. All logs matching this query are analyzed for a potential impossible travel. You can use the `preview` section to see which logs are matched by the current query.
+
+#### User attribute
+
+For the `user attribute`, select the field in the analyzed log that contains the user ID. This can be an identifier like an email address, user name, account identifier.
+
+#### Location attribute
+
+The `location attribute` specifies which field holds the geographic information for a log. It cannot currently be changed and defaults to `@network.client.geoip`, which is the output of the [GeoIP parser][2] logs processor which enriches a log with location information based on the client's IP address.
```

Comment:
> It cannot currently be changed and defaults to

How about something like "The only supported value is `@network.client.geoip` which is enriched by the [GeoIP parser][2]`?